.circleci/config.yml

```
@@ -11,7 +11,8 @@ jobs:
       # Needed to install go
       OS: linux
       ARCH: amd64
-      GOVERSION: 1.13
+      GOVERSION: 1.17
+      GO111MODULE: off
```

LOL... I guess ... yaml?

```
golangci/golangci-lint err this script is deprecated, please do not use it anymore. check https://github.com/goreleaser/godownloader/issues/207
go: unknown environment setting GO111MODULE=false
```

```suggestion
      GO111MODULE: "off"
```

---

Good catch @thaJeztah

---

haha, the good old YAML feature... :)

---

"yes", err, "true"? err.. well, 😂

---

I'll spare some time and avoid thinking how did this even work before ;)

---

Go 1.13/1.14 and before would not use go modules when invoked from inside `GOPATH`. Later versions ignore `GOPATH` and will default to `go modules`.

(but no worries, Golang never has breaking changes, as they have a stability promise 😉 )